src/views/Conference.vue

```diff
@@ -24,23 +24,24 @@ export default {
   props: {
     room: { type: String, default: '' },
   },
+  watch: {
+    // I don't like this approach
```

Comment:
I don't as well

Reply:
the problem that by the Vue router change and the prop change we don't trigget mounted method as before.. If I can understand why that happen, I can make some better solution.

Reply:
@venimus we talked and looks like it's a normal Vue flow - don't trigger mounted at props change. So we have two ways: this one, and the second one is create a separate component that will get that prop and change the router.js.

Reply:
@venimus maybe there is possible to enter the room by jitsi API and remove that all.

Reply:
I was talking about the DOM manipulations actually, but seems you fixed it